        <html><head></head><body>
        <h1>Report for atomicparsley on toolchain
            <a href="index.html">android</a>
        </h1>
        <p>Return code: 1</p>
        <p>Time to run: 00:00:36</p>
        <p>Log output:</p>
        <code><pre><docker run --rm -v /tuscan_data --volumes-from tuscan_data -v /sources --volumes-from sources -v /var/cache/pacman/pkg --volumes-from pkg_cache_android -v logs:/logs make_package_container --sources-directory /sources --shared-directory /tuscan_data atomicparsley
:: Synchronizing package databases...
downloading repo.db...
downloading core.db...
downloading extra.db...
downloading community.db...
Found 1027 packages in cache
===> sudo -u tuscan makepkg --nobuild --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
==> Making package: atomicparsley 0.9.0-5 (Fri Nov 13 23:20:19 UTC 2015)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Downloading AtomicParsley-source-0.9.0.zip...
  -> Found atomicparsley.diff
==> WARNING: Skipping all source file integrity checks.
==> Extracting sources...
  -> Extracting AtomicParsley-source-0.9.0.zip with bsdtar
==> Starting prepare()...
patching file AtomicParsley.cpp
==> Sources are ready.
Copied source directory to /sources/atomicparsley
===> sudo -u tuscan PATH=/tmp/toolchain/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin CC=arm-linux-androideabi-gcc CXX=arm-linux-androideabi-g++ SYSROOT=/build/android-ndk-r10e/platforms/android-21/arch-arm makepkg --noextract --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
Traceback (most recent call last):
  File "/build/make_package.py", line 215, in <module>
    main()
  File "/build/make_package.py", line 212, in main
    result = copy_and_build(args)
  File "/build/make_package.py", line 147, in copy_and_build
    universal_newlines=True)
  File "/usr/lib/python3.5/subprocess.py", line 698, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/usr/lib/python3.5/subprocess.py", line 1055, in communicate
    stdout = self.stdout.read()
  File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xa9 in position 65716: ordinal not in range(128)></pre></code>
        </body></html>
